MkDocs/apm_8.8/docs/matrix.md

```diff
@@ -0,0 +1,100 @@
+   For example, the red cell in the top right corner shows the assets that End of Life is high and Criticality is also High which means high risks.
+   Besides cells, there are also several cards which is summarized count for the different categories of assets, normally High is for assets that need to take action in high priority.
+
+    ![matrix view](./img/matrix_home.png)
+
+2. If there are too many assets in one cells or category, you can also use filter to reduce the counts. The followings are the detailed steps:
+   - click the filter icon, then select `Type`
+   - From the pop-up, find `Substation Transformers`, select it, and press `OK`.  It may be necessary to use the search capability or scroll.
+   - Finally, press `Apply`, the Matrix count 
+
+3. The default matrix version is Criticality and End of life, you can also click the dropdown on the top right corner to change to other matrix versions. 
+
+    ![matrix view](./img/matrix_versions.png)
+
+
+## View the drilled in assets list for one distribution
+`Action:`
+1. Click any cell of Matrix or any card with the blue text of number, you can see the detailed assets list.
+  ![matrix view](./img/matrix_home_click.png)
+  ![matrix view](./img/matrix_drillin.png)
+2. In the drill in page, you can filter out the interested assets list by click the filter icon.
+  ![matrix view](./img/matrix_drillin_filter.png)
+3. there are several actions that can be done in the drill in page:
+   - Click one asset name, and go into asset detail page, create Work order or Service request
+     ![matrix view](./img/matrix_drillin_asset.png)
+     ![matrix view](./img/matrix_asset_details.png)
+   - Add all assets to Asset Investment optimizer project
+     - create a new project
+     - add to existing project
+     ![matrix view](./img/matrix_create_aio.png)
+     ![matrix view](./img/matrix_aio_st.png)
+       * please refer the [Asset Investment Optimization](asset_investment.md) for more details
+   - select some assets by clicking the checkbox and create AIO project
+      ![matrix view](./img/matrix_drillin_select.png)
+
+
+## Config Matrix version
+
+We have by default configuration(include the category cell color/default version etc) for matrix versions, but those configuration can also be changed. For this execise, we will change the cell color.
+
+`Action:`
+1. Click the Config icon in the top right corner
+  ![matrix view](./img/matrix_home_config.png)
+  ![matrix view](./img/matrix_config.png)
+
+2. Click the Edit button under the Categories section
+3. Click the Add button to add a new category 'Medium High' by input the name, color, icon and click the order icon to adjust the sequence.
+  ![matrix view](./img/matrix_config_edit.png)
+4. Click the save button to save the changes
+5. Select the new added category 'Medium High' and then click the cells that should be marked as the orange
```

Comment:
`category`  >  `Matrix cell category color`  Categories show up in other places in the UI should we be specific to say this specific category name.